rosidl_cmake/cmake/rosidl_generate_interfaces.cmake

```diff
@@ -133,6 +133,23 @@ macro(rosidl_generate_interfaces target)
   # afterwards all remaining interface files are .idl files
   list(APPEND _idl_tuples ${_idl_adapter_tuples})
 
+  # to generate action interfaces, we need to depend on "action_msgs"
+  foreach(_tuple ${_interface_tuples})
+    string(REGEX REPLACE ".*:([^:]*)$" "\\1" _tuple_file "${_tuple}")
```

Comment:
The leading `.*` should be removed from the pattern.

Reply:
I only want the second part of the tuple. Without the leading `.*` I will get the full path to the interface file.